bpf/logger/bpf_dbg.h

```diff
@@ -31,7 +31,7 @@ typedef struct log_info {
 
 struct {
     __uint(type, BPF_MAP_TYPE_RINGBUF);
-    __uint(max_entries, 1 << 12);
+    __uint(max_entries, 1 << 15);
```

Comment:
I increased the debugger ringbuffer, because I ended up having to debug with the stdout logs from tests and a lot of messages were missing with 4K. It shouldn't increase the overall footprint, since this is loaded only with OTEL_EBPF_BPF_DEBUG=1